format-text/pom.xml

```diff
@@ -61,6 +61,21 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+      <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
```

Comment:
shouldn't be adding dependencies, junit already exists. In general, junit should also not be a compile scope dependency

Reply:
Removed